code-challenges.js

```diff
@@ -6,18 +6,49 @@ var secretCodeWord1 = "lackadaisical"
 // Expected output: "l4ck4d41s1c4l"
 var secretCodeWord2 = "gobbledygook"
 // Expected output: "g0bbl3dyg00k"
+// turn string into an array
+//iterate through array using .map to equate the values to the expected numbers
+//return the array back to a string using the .join method
+const codedMessage = (string) => {
+  let stringArr = string.split("")
+  let newArr = stringArr.map(value => {
+    if(value == "a"){
+       return value = 4
+    }
+    else if(value == "e"){
+      return value = 3
+    }
+    else if(value == "i"){
+      return value = 1
+    }
+    else if(value == "o"){
+      return value = 0
+    }
+  })
```

Comment:
this solution is SOOO close! the final string is just numbers here... 444114 and 0300. to get the whole _word_ with the replaced letters, you'll need an `else` statement that would `return value` to account for the letters that _aren't_ changing :)